docs/content/latest/quick-start/build-apps/java/ysql-yb-jdbc.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Build a Java application that uses YSQL
+headerTitle: Build a Java application
+linkTitle: Java
+description: Build a sample Java application with the Yugabyte JDBC Driver and use the YSQL API to connect to and interact with YugabyteDB.
+aliases:
+  - /develop/client-drivers/java/
+  - /latest/develop/client-drivers/java/
+  - /latest/develop/build-apps/java/
+  - /latest/quick-start/build-apps/java/
+  - /latest/integrations/jdbc-driver
+menu:
+  latest:
+    parent: build-apps
+    name: Java
+    identifier: java-6
```

Comment:
Need to reorder the pages (identifier) - this should be java-1 so that selecting Java in the left menu displays this page; choosing Java in the menu still brings up the PG driver